README.md

```diff
@@ -17,7 +17,10 @@ If this proposal stands, we plan to extend the [Background Sync](https://wicg.gi
 specification to describe this ability as well, and move the document to the standardization track.
 
 ## Use cases
-  * todo
+  There are two main use cases:
+  1. Web apps can opportunistically update state in the background.
+  2. Periodic content producers like news and Manga sites can push content to
+     users devices, to be consumed at a later, more convenient time.
```

Comment:
The Push API can do this, why is this different?

Reply:
Addressed in the updated text.